### ISSUE TITLE
Loading an episode activates loading of all episodes

### ISSUE THREAD

**Issue by EliasA** [aajanki/yle-dl#318]
Using latest version of yle-dl the behaviour has changed. If I give the URL of the episode of a series I wish to download, yle-dl downloads all episodes of the series and in some cases all episodes of all seasons.

This apparently is because of changes in Areena. Can someone else verify this behaviour and is there something we could do to fix this?

I am happy to help even though I am not familiar with how Areena works in detail.

**Comment by EliasA** [comment]
This seems to have been a temporary situation. I tested this with two different URLs that resulted in behaviour I described.

The first one one was https://areena.yle.fi/1-62142382
and the second one https://areena.yle.fi/1-4402920

Now yle-dl downloads only one episode. Otherwise I would say I have mistakenly used the URL of the landing page of a series but I really followed the progress of the download and saw episode 1 of season 1 being downloaded after episode 1 of season 2 (first example).

My guess is that Yle did something and now they have changed the way things were.
